broker/server.py

```diff
@@ -104,15 +126,17 @@ async def upload_model(file: UploadFile):
         err = traceback.format_exc()
         raise HTTPException(status_code=500, detail=str(err))
 
-
-@app.get("/results/")
+@app.get("/results")
 def download_results():
     services, _, _, _ = read_settings()
     for service in services:
         if "recorder" in service.lower():
-            ip = services[service]["networks"]["custom-network"]["ipv4_address"]
+            host = services[service]["hostname"]
             port = int(services[service]["ports"][0].split(":")[0])
-            url = f"http://{ip}:{port}/download/"
+            
+            url = build_url(host, port, ["download"])    
+            logging.info(f"making a request to url - {url}")
+            
             response = requests.get(url)
             with open(f"{service}.feather", "wb") as out_file:
                 shutil.copyfileobj(response.raw, out_file)
```

Comment:
Until this commit https://github.com/openEDI/oedisi-example/pull/55/commits/b73f3a361c7e8dcaacde8cfa9ef6e6b74e9ab72d is merged, we aren't able to read results:

Reply:
fixed